src/System/Wlog/LoggerNameBox.hs

```diff
@@ -78,6 +78,11 @@ instance (Monad m, HasLoggerName m, Monoid w) => HasLoggerName (RWSStrict.RWST r
 
     modifyLoggerName = RWSStrict.mapRWST . modifyLoggerName
 
+instance HasLoggerName Identity where
+    getLoggerName = mempty
```

Comment:
I would use some meaningful name here, at least `Identity`. Otherwise someone can accidentally use this instance and not understand what's going on.